Tests/PositionDropBoxDataTest.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+require_once 'src/PositionDropBoxData.php';
+require_once 'src/Entities/PlayerCard.php';
+
+
+use PHPUnit\Framework\TestCase;
+
+class PositionDataTest extends TestCase
```

Comment:
This test is throwing a warning that \`1) Class PositionDropBoxDataTest cannot be found in /var/www/html/collections/matt/Tests/PositionDropBoxDataTest.php  
\`